scylla/src/frame/response/cql_to_rust.rs

```diff
@@ -68,6 +68,10 @@ macro_rules! impl_from_cql_val {
 }
 
 impl_from_cql_val!(i32, as_int); // i32::from_cql<CQLValue>
+impl_from_cql_val!(u32, as_date); // u32::from_cql<CQLValue>
+impl_from_cql_val!(f32, as_float); // i32::from_cql<CQLValue>
+impl_from_cql_val!(f64, as_double); // u32::from_cql<CQLValue>
+impl_from_cql_val!(bool, as_boolean); // u32::from_cql<CQLValue>
```

Comment:
Please change `u32`/`i32` in the comments to correct types.

Reply:
Fixed, I must have left it by mistake.